modules/gdscript/gdscript_vm.cpp

```diff
@@ -123,6 +123,31 @@ static String _get_var_type(const Variant *p_var) {
 }
 #endif // DEBUG_ENABLED
 
+Variant GDScriptFunction::_get_default_variant_for_data_type(const GDScriptDataType &p_data_type) {
+	if (p_data_type.kind == GDScriptDataType::BUILTIN) {
+		if (p_data_type.builtin_type == Variant::ARRAY) {
+			Array array;
+			// Typed array.
+			if (p_data_type.has_container_element_type()) {
+				const GDScriptDataType &element_type = p_data_type.get_container_element_type();
```

Comment:
```
modules/gdscript/gdscript_vm.cpp:132:29: error: unused variable 'element_type' [-Werror,-Wunused-variable]
                                const GDScriptDataType &element_type = p_data_type.get_container_element_type();
                                                        ^
```

Reply:
Yeah, mistake on my part. Fixing it properly now